neighsyncd/neighsyncd.cpp

```diff
@@ -27,16 +29,21 @@ int main(int argc, char **argv)
             NetLink netlink;
             Select s;
 
-            netlink.registerGroup(RTNLGRP_NEIGH);
-            cout << "Listens to neigh messages..." << endl;
-            netlink.dumpRequest(RTM_GETNEIGH);
-
-            s.addSelectable(&netlink);
             if (sync.getRestartAssist()->isWarmStartInProgress())
```

Comment:
Yes, the process for that is the same. only difference is that the restore_neighbors service will restore the table to kernel in case of warm reboot case.